src/env.sh

```diff
@@ -64,6 +64,9 @@ env_init_pkg() {
         PKG_PATH="$ELLIPSIS_PACKAGES/$PKG_NAME"
     fi
 
+    # Skip non directory packages
+    [ -d "$PKG_PATH" ] || return 0
```

Comment:
This is already handled by the check on line 102, in the else clause;

https://github.com/ellipsis/ellipsis/pull/110/files#diff-507cfddc4b1a41a598606eb25c8b80fd63afb3b34bb147d13060c97698f39b0fR99-R104

The idea is that there shouldn't be any loose files in the package directory, if they are the current behavior in the else will ignore them, but also log an error so we at least know something fishy is going on. (Which might cause issues for other functions as well)

Reply:
Hey, good catch, and indeed this patch does not make that much sense in the context of that PR. The idea of this line was to not throw an error in case of loose files in the package directory, but as you mentioned, it should not happen. So we can definitely remove this one.

For the full story, I assume this patch comes from another future PR I'd like to propose later. My dev workflow has been quite messy, and I'm trying to figure out to improve it. So, my mistake here